metals/src/main/scala/scala/meta/internal/metals/ServerLivenessMonitor.scala

```diff
@@ -122,13 +115,13 @@ class ServerLivenessMonitor(
     TimeUnit.MILLISECONDS,
   )
 
-  def isBuildServerResponsive: Boolean = isServerResponsive
+  def isBuildServerResponsive: Boolean = bspStatus.isBuildServerResponsive
 
   def shutdown(): Unit = {
     scribe.debug("shutting down server liveness monitor")
     scheduled.cancel(true)
     scheduler.shutdown()
-    client.metalsStatus(ServerLivenessMonitor.disconnectedParams)
+    bspStatus.connected()
```

Comment:
```suggestion
    bspStatus.disconnected()
```
? Since this is on shutdown? Seems `disconnected` is not used otherwise?